### DISCUSSION TITLE
[UI] Negative Prompt: smaller and scrollable (or resizable) textarea

### DISCUSSION THREAD

**Discussion by Tillerz** [vladmandic/sdnext#446] [Ideas]
Can we have this smaller or resizable? Usually the "negative prompt" field contains a lot of entries, making the whole text2image page larger than the screen so between GENERATE and SAVE you always have to scroll.

Options that come into my mind:

- smaller field that scrolls
- resizable field
- smaller (or customizable) font for that one field

I don't want to zoom out the whole browser window because then lots of stuff is hard to read (not everyone is in their 20s anymore).

**Comment by gobshyte** [comment]
I also find it a pain if either:
a) you have lots of loras (etc.) open
b) you are on a mobile device/in portrait/narrow web browser

Sometimes it is really bad, you have to scroll all way to the top  of the page, press Generate, scroll all the way to the end, check the image, scroll all the way back up, etc. etc.

I think you can move some of the items on the left of txt2img around, but I could not see a way to move the generate button so it is adjacent to the image output. Would be really nice if you ould move it, or if it was next to it in when in a narrow window.
